lstchain/reco/r0_to_dl1.py

```diff
@@ -661,7 +661,8 @@ def r0_to_dl1(
 
                 # Muon ring analysis, for real data only (MC is done starting from DL1 files)
                 if not is_simu:
-                    bad_pixels = event.mon.tel[telescope_id].calibration.unusable_pixels[0]
+                    bad_pixels = source.r0_r1_calibrator.mon_data.tel[telescope_id].calibration.unusable_pixels[0]
```

Comment:
Code would be much shorter and reable if you store this once into a variable and then use it:

```
calibration_mon = source.r0_r1_calibrator.mon_data.tel[telescope_id].calibration
```